src/chain/public-key.ts

```diff
@@ -68,6 +68,14 @@ export class PublicKey implements ABISerializableObject {
         this.data = data
     }
 
+    /**
+     * Returns the core 33-byte compressed public key data as a Uint8Array.
+     * This is suitable for cryptographic operations like verification.
+     */
+    getCompressedKeyBytes(): Uint8Array {
+        return this.data.array.subarray(0, 33)
```

Comment:
Just so the result is mutable, I would probably do something like this instead:

```suggestion
        return new Uint8Array(this.data.array.subarray(0, 33))
```

Reply:
I don't think one would need the result as mutable.

Reply:
Yeah, you are right, we should make the code idiot proof. I didn't realize that you can unintentionally mutate the original Uint8Array while mutating the returned value. I mean developers wouldn't need to mutate the returned value, which is functionally meaningless, but you are right, what if he made a mistake.

Reply:
Since the PR is merged, i added another PR to switch to use `slice`, i think it will solve the issue.
https://github.com/wharfkit/antelope/pull/117